app/Http/Controllers/SmartXpScreenController.php

```diff
@@ -114,7 +98,9 @@ public function smartxpTimetable()
     /** @return View */
     public function canWork()
     {
-        return view('smartxp.caniwork', ['timetable' => $this->smartxpTimetable()->roster,
-            'occupied' => $this->smartxpTimetable()->occupied, ]);
+        $timetable = $this->smartxpTimetable();
+
+        return view('smartxp.caniwork', ['timetable' => $timetable->roster,
+            'occupied' => $timetable->occupied, ]);
```

Comment:
Does this even work right now now that the university has switched to TimeEdit?

Reply:
Yes, we sync timedit via the ical link to our own google calendar